fenicsxconcrete/boundary_conditions/boundary.py

```diff
@@ -98,7 +104,7 @@ def boundary(x):
     return boundary
 
 
-def show_marked(domain, marker):
+def show_marked(domain, marker):  # pragma: no cover
```

Comment:
it can be used to show the nodes associated with the marker, to debug that you defined your boundary correctly.
I wasn't sure how to test this because its just plotting something. Not sure if this is a good reason, since we could also change it, to make it coverable. What do you think?
Cover it or just add some comments?

Reply:
I would add the comments either way.
I have no strong feeling about the coverage here. This is not really part of our module.
However if there is an easy way to also test this function, I would say, lets do it. This way we would ensure that even this feature is working.
The simplest way I can think of is, instead of a `plt.show()` just save a jpg file. Or is there a simple way to execute plt.show() in the CI?

Reply:
> The simplest way I can think of is, instead of a plt.show() just save a jpg file.

I think so too. I will think about it.